pkg/dbsyncers/subscription_status_syncer.go

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package dbsyncers
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"k8s.io/apimachinery/pkg/api/errors"
+	appsv1 "open-cluster-management.io/multicloud-operators-subscription/pkg/apis/apps/v1"
+	appsv1alpha1 "open-cluster-management.io/multicloud-operators-subscription/pkg/apis/apps/v1alpha1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	subscriptionsSpecTableName          = "subscriptions"
+	subscriptionStatusesStatusTableName = "subscription_statuses"
```

Comment:
the variable name is a bit strange.
isn't subscriptionStatusesTableName is clear enough?